src/protocols/fp-car.ts

```diff
@@ -0,0 +1,137 @@
+// import { Result } from "@adviser/cement";
+import { FPEnvelopeTurnaround, FPEnvelopeType, FPStoreType } from "./fp-envelope";
+import { CID } from "multiformats";
+
+export interface FPGetCarReq {
+    readonly cid: CID;
+}
+/*
+ * A request to get a CAR file.
+ * As JSON, this is:
+ * {
+ *    "type": "get-car-req",
+ *    "tid": "123",
+ *    "payload": {
+ *       "cid": "bafybeib2v"
+ *    }
+ *    "auth": { ... }
+ *    "meta": { ... }
+ * }
+ */
+export interface FPEnvelopeGetCarReq extends FPCarReq<FPGetCarReq> {
+    readonly type: FPEnvelopeType.GET_CARREQ;
+    readonly storeType: FPStoreType.CAR;
+}
+
+export interface FPGetCarRes {
+    readonly cid: CID;
+    readonly bytes: Uint8Array;
+}
+/*
+ * The response to a GetCar request.
+ * As JSON, this is:
+ * {
+ *    "type": "get-car-res",
+ *    "tid": "123",
+ *    "payload": {
+ *       "cid": "bafybeib2v"
+ *       "bytes": [0x65, 0x66, 0x67, ...],
```

Comment:
why not just send the car raw as binary? the CAR get and PUT dont need to be parsed by the server